ee/rpc/audsrv/include/audsrv.h

```diff
@@ -221,9 +221,11 @@ int audsrv_adpcm_init();
 /** Sets output volume for the specified voice channel.
  * @param ch       Voice channel ID
  * @param vol      volume in percentage (0-100)
+ * @param pan      left/right offset [-100 .. 0 .. 100]
  * @returns 0 on success, negative otherwise
  */
-int audsrv_adpcm_set_volume(int ch, int vol);
+int audsrv_adpcm_set_volume_and_pan(int ch, int vol, int pan);
+#define audsrv_adpcm_set_volume(ch, vol) audsrv_adpcm_set_volume_and_pan(ch, vol, 0) //For backward-compatibility
```

Comment:
Make this a regular function with the same signature as before?

```c
int audsrv_adpcm_set_volume(int ch, int vol);
```

Reply:
Hmm I think it's better to stick to the same pattern that has been used with other functions that changed.

Reply:
The 2 functions here have different names and the non-pan version is still a useful function, it's not just for backwards compatibility.

Reply:
how so? withe pan set to zero it gives the same result as the old one

Reply:
Well it's probably nice to not have to pass zero when you don't need to pan but either way is fine by me